src/views/Hearing/index.js

```diff
@@ -225,12 +226,14 @@ function groupSections(sections) {
     const sectionGroup = find(sectionGroups, group => section.type === group.type);
     if (sectionGroup) {
       sectionGroup.sections.push(section);
+      sectionGroup.n_comments += section.n_comments;
```

Comment:
Right. I must've been spectacularly out of it while trying to review this. Is fine.